target/linux/ramips/image/rt288x.mk

```diff
@@ -47,13 +47,13 @@ endef
 TARGET_DEVICES += dlink_dap-1522-a1
 
 define Device/f5d8235-v1
+  DTS := F5D8235_V1
   IMAGE_SIZE := 7744k
   DEVICE_TITLE := Belkin F5D8235 V1
   DEVICE_PACKAGES := kmod-switch-rtl8366s kmod-usb-core kmod-usb-ohci \
     kmod-usb-ohci-pci kmod-usb2 kmod-usb2-pci kmod-usb-ledtrig-usbport
-  DEVICE_DTS := F5D8235_V1
-  KERNEL := kernel-bin | append-dtb | lzma | uImage lzma
-  KERNEL_INITRAMFS := kernel-bin | append-dtb | lzma | uImage lzma
+  KERNEL := $(KERNEL_DTB) | uImage lzma
```

Comment:
I'm quite sure you can drop `KERNEL` and `KERNEL_INITRAMFS` here. In `target/linux/ramips/image/Makefile` we have already:

```
define Device/Default
  ...
  KERNEL := $(KERNEL_DTB) | uImage lzma
  ...
endef
```

And in `include/image.mk` we have `KERNEL_INITRAMFS = $$(KERNEL)`.